vcsh

```diff
@@ -56,7 +56,8 @@ shift $((OPTIND-1))
 
 source_all() {
 	# Source file even if it's in $PWD and does not have any slashes in it
-	case $1 in
+	# shellcheck source=/dev/null
```

Comment:
What does this mean?

Reply:
To fully parse things, shellcheck needs to follow `. some.file.sh` lines. If the file doesn't exist, you can point it at one. I cheat here and say "ignore this" by feeding it `/dev/null`.